editoast/osm_to_railjson/justfile

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env just --justfile
+
+# Display available recipes
+help:
+    @just --list --unsorted
+
+# Run the application; does not start other services
+run *args:
+    cargo run {{args}}
+
+# Install required dependencies
+install:
+    cargo build
+
+# Run the tests
+test:
+    cargo test
+
+# Format the source code
+format:
+    cargo fmt
+
+# Verify code lints without applying any fix
+lint:
+    cargo clippy --all-features
```

```suggestion
    cargo clippy --workspace --all-features --all-targets
```

And this should also be spread to `fix-lint` below (whether workspace and all-targets are in or not)

---

I don’t think that we want a --workspace here. That triggers a check on all editoast, not just osm_to_railjson